BinaryGap/Solution.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace BinaryGap;
+
+public class Solution
+{
+    public string Binary { get; set; }
+    public int solution(int N)
```

Comment:
rename method's name by PascalCase convention and parameter to camelCase and apply everywhere
you can checkout conventions for C# 
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions